app/Http/Controllers/Admin/CategoriesController.php

```diff
@@ -40,17 +40,27 @@ public function create()
     public function store(Request $request)
     {
         $validator = Validator::make($request->all(), [
-            'namecategory' => 'required|regex:/^([a-zA-Z0-9ÀÁÂÃÈÉÊÌÍÒÓÔÕÙÚĂĐĨŨƠàáâãèéêìíòóôõùúăđĩũơƯĂẠẢẤẦẨẪẬẮẰẲẴẶẸẺẼỀỀỂưăạảấầẩẫậắằẳẵặẹẻẽềềểỄỆỈỊỌỎỐỒỔỖỘỚỜỞỠỢỤỦỨỪễệỉịọỏốồổỗộớờởỡợụủứừỬỮỰỲỴÝỶỸửữựỳỵỷỹ\s]+)$/'
+            'namecategory' => 'required|regex:/^([a-zA-Z0-9ÀÁÂÃÈÉÊÌÍÒÓÔÕÙÚĂĐĨŨƠàáâãèéêìíòóôõùúăđĩũơƯĂẠẢẤẦẨẪẬẮẰẲẴẶẸẺẼỀỀỂưăạảấầẩẫậắằẳẵặẹẻẽềềểỄỆỈỊỌỎỐỒỔỖỘỚỜỞỠỢỤỦỨỪễệỉịọỏốồổỗộớờởỡợụủứừỬỮỰỲỴÝỶỸửữựỳỵỷỹ\s]+)$/',
+            'filename1' => 'required|image|mimes:jpeg,png,jpg,gif,svg|max:2048',
```

Comment:
+ Chỗ ni filename1 bị sai tên. Bên blade đang để filename
+ filename truyền lên đang là cái mảng nên không validate theo kiểu ni được. Kiểu ni đang validate cho 1 file image.
=> Suggest: https://stackoverflow.com/questions/31534413/image-array-validation-in-laravel-5 
Dựa theo cái này để sửa code